src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -608,11 +616,14 @@ def __init__(self, module_inst, fscid):
         self.fscid = fscid
 
     @ViewCache()
-    def get(self):
+    def get(self, suppress_errors='True'):
         try:
             ret = CephService.send_command('mds', 'session ls', srv_spec='{0}:0'.format(self.fscid))
         except RuntimeError:
-            ret = []
+            if ast.literal_eval(suppress_errors):
+                ret = []
+            else:
+                raise
```

Comment:
I'm not gonna lie this was wild to see using ast. Compiler nerd me is happy. Nevertheless, I think it would be better to use the boolean `True` value so we don't auto impale ourselves in the future.

Reply:
sure